plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -754,7 +756,7 @@ void GSRendererDX11::EmulateTextureSampler(const GSTextureCache::Source* tex)
 
 	m_ps_sel.ltf = bilinear && shader_emulated_sampler;
 
-	m_ps_sel.point_sampler = !bilinear || shader_emulated_sampler;
+	m_ps_sel.point_sampler = dev->m_gpu_vendor == GSDevice11::GPU_VENDOR::VENDOR_AMD && (!bilinear || shader_emulated_sampler);
```

Comment:
I think it would be better to define `m_gpu_member` as a protected member of `GSRendererHW`. You can write some nice utility function like `bool isAMD()` to make this code a bit nicer. Also OGL has a way to get vendor information. See https://www.khronos.org/opengl/wiki/OpenGL_Context#Context_information_queries

Maybe it has no use for OGL now but I think it's simple to add and won't hurt anything. @turtleli thoughts?

Reply:
We already have hacks and checks around GL_VENDOR. 
It's kinda useless alone though, since driver version is usually the most important element #2696

Reply:
Yeah we have vendor checks in GLLoader so it has no use for gl, with that in mind putting it in gsdevice now someone might use the wrong vendor check for gl.

Reply:
The idea is that it should be generic. All renderers have a way to check vendor. There's no reason we need 2-3 different ways to check depending on renderer. Implementation details shall be the responsibility of the respective renderers.